rules/linux/file_event/file_event_lnx_wget_download_file_in_tmp_dir.yml

```
@@ -0,0 +1,27 @@
+title: Wget Creating Files in Tmp Directory
+id: 35a05c60-9012-49b6-a11f-6bab741c9f74
+status: experimental
+description: Detects the use of wget to download content in a temporary directory such as "/tmp" or "/var/tmp"
+references:
+    - https://blogs.jpcert.or.jp/en/2023/05/gobrat.html
+    - https://jstnk9.github.io/jstnk9/research/GobRAT-Malware/
+    - https://www.virustotal.com/gui/file/60bcd645450e4c846238cf0e7226dc40c84c96eba99f6b2cffcd0ab4a391c8b3/detection
+    - https://www.virustotal.com/gui/file/3e44c807a25a56f4068b5b8186eee5002eed6f26d665a8b791c472ad154585d1/detection
+author: Joseliyo Sanchez, @Joseliyo_Jstnk
+date: 2023/06/02
+tags:
+    - attack.command_and_control
+    - attack.t1105
+logsource:
+    product: linux
+    category: file_event
+detection:
+    selection:
+        Image|endswith: '/wget'
+        TargetFilename|startswith:
+            - '/tmp/'
+            - '/var/tmp/'
+    condition: selection
+falsepositives:
+    - Legitimate downloads files in tmp folder.
+level: high
```

```suggestion
level: medium
```
downloading stuff to tmp is also just typical admin behaviour